ppx

```diff
@@ -0,0 +1,13 @@
+# Get the directory of the script to work from.
```

Comment:
This should have a shebang: `#!/usr/bin/env bash`.
I actually just had a problem with this when trying to use `bs-let` on Travis; it gives me this error:
```
/home/travis/build/ryb73/prom-ex/node_modules/bs-let/ppx: 7: [: Linux: unexpected operator
/home/travis/build/ryb73/prom-ex/node_modules/bs-let/ppx: 10: [: Linux: unexpected operator
```

Reply:
Oh, interesting. I haven't tried deploying with this yet, only using it locally. What shell is running on Travis? Bash?

Reply:
I found on this repo:
https://github.com/chartmuseum/helm-push/issues/7

that it looks like using a double equals causes issues on some shells. I'll make the change on bs-let as well.